specification/migrate/resource-manager/Microsoft.OffAzure/stable/2020-07-07/migrate.json

```diff
@@ -2799,16 +2938,19 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/MasterSite"
+          "202": {
+            "description": "Accepted",
```

Comment:
This looks wrong to me. Now, there's no way to tell what the shape of the eventual response is going to be. I believe all other resources declare their final response, even if in practice it's a long running operation, similarly to what the previous version did.